test/test_examples.sh

```diff
@@ -9,15 +9,16 @@ SOURCE_DIR=$1
 BUILD_DIR=$2
 INSTALL_DIR=$3
 CMAKE_INSTALL_PREFIX=$4
+STANDALONE_CMAKE_OPTIONS=$5
 
 echo "Running: $0 $*"
 
 function print_usage() {
 	echo "$(basename $0) - test all examples standalone"
-	echo "Usage: $(basename $0) <source_dir> <build_dir> <install_dir> <CMAKE_INSTALL_PREFIX> <list-of-examples-to-run>"
+	echo "Usage: $(basename $0) <source_dir> <build_dir> <install_dir> <CMAKE_INSTALL_PREFIX> <standalone_cmake_options> <list-of-examples-to-run>"
 }
 
-if [ "$5" = "" ]; then
+if [ "$6" = "" ]; then
```

Comment:
non-blocking comment: I hope that this script will one day be written in Python

Reply:
I hope so ;-)

BTW. This is a blocking comment, since:
- @bratpiorka requested changes
- Merging is blocked: Merging can be performed automatically once the requested changes are addressed.